### ISSUE TITLE
[Bug]: Ashford [UK] - Page layout changed, parser fails

### ISSUE THREAD

**Issue by Mustek** [mampfes/hacs_waste_collection_schedule#1914]
### I Have A Problem With:

A specific source

### What's Your Problem

Council has updated their page layout slightly after contracting a new waste collector.

The `table` the parser is looking for is no longer present. All data is still available on the page.

### Source (if relevant)

ashford_gov_uk

### Logs

```Shell
failed for source Ashford Borough Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/ashford_gov_uk.py", line 85, in fetch raise Exception( Exception: could not get valid data from ashford.gov.uk. is your UPRN (<UPRN REMOVED>) correct for postcode (<POSTCODE REMOVED>)? check https://uprn.uk/<UPRN REMOVED> and https://secure.ashford.gov.uk/wastecollections/collectiondaylookup/
```

### Relevant Configuration

```YAML
# Waste Collection
waste_collection_schedule:
    sources: 
    - name: ashford_gov_uk
      args: 
        uprn: <UPRN REMOVED>
        postcode: <POSTCODE REMOVED>
      customize:
        - type: Recycling (green bin / clear sacks)
          alias: Recycling
        - type: Food waste (black container with orange lid)
          alias: Food Waste
        
sensor:
    - platform: waste_collection_schedule
      name: ashford_gov_uk
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by Mustek** [comment]
NB: I would create a PR myself, but my test environment is having some issues between Python 3.11 and OpenSSL 3.2.1 on MacOS, not allowing me to test the source.

**Comment by 5ila5** [comment]
I have some weird SLL issues with multiple British providers as well. Need to look into a fix/workaround.

**Comment by ChopsKingsland** [comment]
I can have a look into it properly tomorrow. I’ve been playing around with it tonight as just a python script by itself as I would like to integrate into another project

**Comment by ChopsKingsland** [comment]
From a quick look this morning, it seems as if the tables are still there (one for each bin type).

Just before the error is thrown, if you get it to print out the HTML contents of `soup`, that gives out the page with the select address dropdown still present and no bin data - it looks like the form isn't being filled in correctly.

**Comment by wormvortex** [comment]
Did anyone work out a fix for this issue?

**Comment by SHRMark** [comment]
Any work arounds for this? I have time on the weekend to have a look too.

**Comment by ChopsKingsland** [comment]
I had a look but couldn’t work out what was going wrong - the online form is a weird way to get bin data

**Comment by 5ila5** [comment]
Fixed in #2040 

This will be part of release 1.48.0 or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Comment by SHRMark** [comment]
It looks like the council made another change at the same time which is throwing up “unknown” on the collection date. The web page uses “today” instead of a date when the collection is due that day and that returns a null for that bin.

**Comment by 5ila5** [comment]
@SHRMark fixed in #2103
